pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -1262,8 +1262,21 @@ func (l *LibvirtDomainManager) FreezeVMI(vmi *v1.VirtualMachineInstance, unfreez
 func (l *LibvirtDomainManager) UnfreezeVMI(vmi *v1.VirtualMachineInstance) error {
 	l.cancelSafetyUnfreeze()
 	domainName := api.VMINamespaceKeyFunc(vmi)
-	// fs thaw is idempotent by itself
-	_, err := l.virConn.QemuAgentCommand(`{"execute":"guest-fsfreeze-thaw"}`, domainName)
+	cmdResult, err := l.virConn.QemuAgentCommand(`{"execute":"`+string(agentpoller.GET_FSFREEZE_STATUS)+`"}`, domainName)
```

Comment:
Does this have the same semantics in Windows as in Linux?  I assume so but we should confirm.

Reply:
This question is relevant to any qemu agent command so I guess it should work. don't we have windows tier2 tests with online snapshot? @dalia-frank can you help with answering that?